spec/controllers/admin/base_controller_spec.rb

```diff
@@ -9,9 +9,6 @@ def index
 
   let(:account) { create :account }
 
-  before do
-    allow(controller).to receive(:setup_request)
```

Comment:
I guess this could go to another PR or commit at least, not directly related to `synced` I think.